src/Common/HashTable/BloomFilt.h

```diff
@@ -0,0 +1,118 @@
+#pragma once
+
+#include <functional>
+#include <cmath>
+//#include <MurmurHash3.h>
+//#include <Functions/FunctionsHashing.h>
+#include <city.h>
+#include <random>
+#include <Common/HashTable/HashTableAllocator.h>
+
+template <typename T, typename Hash, typename Allocator = HashTableAllocator>
+class BloomFilt : protected Hash,
```

Comment:
Can we reuse (maybe partially) `BloomFilter` from 

https://github.com/ClickHouse/ClickHouse/blob/f77cc8946466af0faa3e82f88b2ccfaf44ee1a62/src/Interpreters/BloomFilter.h#L27-L34

?

Have you missed this implementation or new one have some important differences? If existing doesn't fit, than the improvements may be described in a comment.